align/gdsconv/gen_pl_gds.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
```

Comment:
How is this related to ILP placer?

Reply:
It is not relevant. Just a script to visualize the placement as GDS. I'll remove it since a better version is already in master now.

Reply:
If you still want to keep this, you can move it to dev/Experimental area so that it does not impact coverage.